next.config.js

```diff
@@ -8,19 +9,22 @@ const withMDX = require('@next/mdx')({
     remarkPlugins: [slug, [toc, { maxDepth: 2 }], gfm]
   }
 })
-module.exports = withMDX({
-  pageExtensions: ['tsx', 'js', 'jsx', 'mdx', 'ts'],
-  env: {
-    DB_NAME: process.env.DB_NAME || 'c0d3dev6',
-    DB_USER: process.env.DB_USER || 'c0d3db',
-    DB_PW: process.env.DB_PW || 'letmein2',
-    DB_PORT: process.env.DB_PORT || '5432',
-    DB_HOST: process.env.DB_HOST || 'freedomains.dev',
-    SENTRY_DSN:
-      process.env.SENTRY_DSN ||
-      'https://e95626afb0454145b569bc69116f838c@o385150.ingest.sentry.io/5221680',
-    POSTHOG_API_KEY: process.env.POSTHOG_API_KEY,
-    SESSION_SECRET: process.env.SESSION_SECRET || 'c0d3hard3r',
-    SERVER_URL: process.env.SERVER_URL || '/api/graphql'
-  }
+const moduleExports = withMDX({
+  pageExtensions: ['tsx', 'js', 'jsx', 'mdx', 'ts']
 })
+
+const SentryWebpackPluginOptions = {
+  // Additional config options for the Sentry Webpack plugin. Keep in mind that
+  // the following options are set automatically, and overriding them is not
+  // recommended:
+  //   release, url, org, project, authToken, configFile, stripPrefix,
+  //   urlPrefix, include, ignore
+  debug: false,
+  silent: true // Suppresses all logs
```

Comment:
Could you add a short comment explaining why do we need to supress logs?

Reply:
how does this sound 
```
  // This plugin is used to send source map files to sentry.io during build time.
  // Silencing the log hides all the information about each individual file being
  // sent keeping the build logs cleaner and still logging any error that occurs.
  silent: true
```